docs/governance/templates/release_issue.md

```diff
@@ -79,7 +78,7 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] If full release move [helm `tag` value][values] is set to {version}+1-dev
 - [ ] If full release move the [helm `Chart` version values][chart] is to {version}+1-dev
 - [ ] If full release move the [`sdks/nodejs/package.json`][nodejs] to {version}+1-dev
```

Comment:
I think this is the correct step but we should replace with the following command
```
cd sdks/nodejs
npm version preminor --preid=dev
```
I.e. change to:
```
- [ ] If full release, change to the `sdks/nodejs` directory and run the command `npm version preminor --preid=dev` to update the package version
```

Reply:
That formats the version slightly differently, e.g.

`version": "1.16.0-dev.0"` instead of `version": "1.16.0-dev`

Reply:
Removing the trailing `.0` does not seem to be supported by npm and I think is due to how semver is handled internally (and by the semver package)
https://docs.npmjs.com/cli/v6/commands/npm-version
https://github.com/npm/node-semver

I think it is this way to support publishing multiple dev versions prior to a release, because it is not possible to re-publish an existing version. Perhaps idiomatic Node.js

An alternative would be specify the version explicitly rather than rely on the increment feature of npm - may be better for this project?
```
npm version 1.16.0-dev
```

That seems clearer

I'm hoping we can avoid the situation where we manually edit `package.json` as we must then run `npm install` to update `package-lock.json`

Reply:
Marking as resolved, since I've moved this comment thread over to #2120.